### DISCUSSION TITLE
Can't run nightly on Windows

### DISCUSSION THREAD

**Discussion by p4thie** [wezterm/wezterm#3833] [Q&A] [answered]
I cannot launch wezterm nightly on Windows.

Tried both `setup.exe` and `zip`. For each cases, launching `wezterm.exe` results in the Windows dialog saying "wezterm.exe stopped working", and launching `wezterm-gui.exe` does nothing (no window or whatsoever showsup).

For the first case (where "wezterm.exe stopped working" window pops up) I tried to get more information by debugging with Visual Studio. The debugger says

> Unhandled exception at 0x00007FFDE96BC9F0 (igc64.dll) in wezterm-gui.exe: 0xC0000005: Access violation reading location 0x0000015A0000011A.

I thought this could be a privilege issue so I also tried running as admin, but got the same result.
Is there anyone having similar issues, or is it just me?
Non-nightly version works just fine. I'm on Windows 11, version 10.0.22621 Build 22621.

**Comment by wez** [comment, marked as answer]
try setting `front_end='OpenGL'` in the config, and please file a bug report issue for this so that it can be tracked appropriately

**Reply by p4thie** [reply]
thank you for the prompt response. setting `front_end='OpenGL` in the config solved the issue. Since this is not an expected behavior, I filed a bug report issue below. I'll mark your comment as answer so that we move this discussion to the issues.
https://github.com/wez/wezterm/issues/3834
